src/index.css

```diff
@@ -2,14 +2,21 @@
 
 :root {
   --primary-color: #3b5bdb;
-  --secondary-color: #ffffff;
-  --tertiary-color: #e9ecef;
+  --background-color: #ffffff;
+  --bright-border-color: #efefef;
+  --body-color: #e9ecef;
   --accent-color: #364fc7;
-  --faded-txt: #696969;
-  --snippet1: #a0f4dc;
-  --snippet2: #fcd3d4;
-  --danger: #ff0000;
-  --success: #008000;
+
+  --bright-txt-color: #ffffff;
+  --dark-txt-color: #000000;
+  --faded-txt-color: #696969;
+
+  --danger-color: #ff0000;
+  --success-color: #008000;
+
+  /* Event colors */
+  --conference-color: #a0f4dc;
```

Comment:
Siin võiks mõelda vb isegi mingi prefixi peale. Näiteks --event-conference-color. Siin on kommentaar küll ilusti olemaas üleval, aga kui seda variable-it kasutada mujal failides, siis see kommentaar pole seal nähtav.